### DISCUSSION TITLE
Easily Customize the Delete Resource Modal

### DISCUSSION THREAD

**Discussion by ekazda** [laravel/nova-issues#6137] [Ideas & Feature Requests]
It would be great if there were an easy way to simply pass values into the 'Delete Resource' modal to update both the title and the content. I would prefer if the title weren't generic. Additionally, I would like to include more information about what the delete action is actually deleting, especially if there are any cascading deletes occurring.

![Screenshot 2024-01-08 at 10 05 59 PM](https://github.com/laravel/nova-issues/assets/1107804/6fa92e22-8405-4163-a69b-04ec164f8e26)

**Comment by HugoHeneault** [comment]
And with this it would be perfect: https://github.com/laravel/nova-issues/discussions/5403
